src/Makefile.OCaml

```diff
@@ -51,6 +51,10 @@ OCAMLLIBDIR:=$(shell $(OCAMLC) -config-var standard_library)
 #
 OCAMLFIND := $(shell command -v ocamlfind 2> /dev/null)
 
+# Any invocation of the non-Xcode version of xcodebuild will trigger an error message
+# and an non 0 exit code. 
```

Comment:
This doesn't in my view unconfuse people who don't already understand.  The idea that there is a non-Xcode version or codebuild is surprising.   I'd like to see that explained (so that anyone maintaining the makefile will be enlightened).

Reply:
Yes, that's better.  The basic issue is that many people, including people that build open source code on macs, are unclear on the twisty maze of CLT and XCode and how it targets various SDKs.   It doesn't help that CLT installs an xcode-build, or that you seem to have to install XCode but not install it to get CLT.    Ideally this would be well-explained someplace on the net and we could point to it in INSTALL - that's not about this PR, though.

Reply:
Ok thanks you.